styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -1,3 +1,98 @@
+.rsHomepage-announcement {
+  display: flex;
+  margin-top: -70px;
+  width: 100%;
+}
+
+.rsHomepage-annLeft {
+  background-image: url('/imgs/home-pageAnn-left.png');
+  background-position: top left;
+  background-repeat: no-repeat;
+  background-size: cover;
+  float: left;
+  min-height: 450px;
+  padding-left: 0;
+  padding-right: 0;
+  width: 100%;
+  z-index: 2;
+}
+
+.rsHomepage-annLeftContainer {
+  float: left;
+  margin-top: 80px;
+  min-height: 10px;
+  width: 100%;
+}
+
+.rsHomepage-annLeftBar {
+  background-color: $gray-darkest;
+  padding: 10px 35px;
+  width: 100%;
+}
+
+.rsHomepage-annLeftBar-header {
+  color: $gray-lighter;
+  margin: 0;
+}
+
+.rsHomepage-annLeftBar-subHeader {
+  color: $gray-lighter;
+  margin: 0;
+}
+
+.rsHomepage-annLeft-content {
+  color: $gray-dark;
+  min-height: 10px;
+  padding: 35px;
+  width: 100%;
+}
+
+.rsHomepage-annLeft-awardsRow {
+  @include make-row;
+  padding-top: 20px;
+}
+
+.rsHomepage-annRight {
+  display: none;
+}
+
+.rsHomepage-annRight-iconBar {
+  background-color: $rackspace-red;
+  color: $white;
+  float: left;
+  line-height: 1.6;
+  padding: 0;
+  text-align: center;
+  width: 85px;
+
+  &::before {
+    content: '\e983';
+    font-family: 'rswebfonts';
+    font-size: 50px;
+  }
+}
+
+// Desktop only rules
+@media (min-width: $screen-md-min) {
+
+  .rsHomepage-annRight {
+    background-image: url('/imgs/home-pageAnn-right.png');
```

Comment:
Also needs to be relative path.

Reply:
![image](https://user-images.githubusercontent.com/5263371/30069158-81d495d2-9225-11e7-8047-1d3698dc9036.png)
I'm glad this is loading on desktop only, but this image is 799KB and takes a whole second to load. This seems like an unnecessary performance burden for a small component, especially given that this is for our homepage. 

Are there CSS-only mechanisms for making these angles instead of relying on images?

Reply:
I have tried css clip-path and skew to get the angel but with too many cons. The clip-path style we need does not work in FF or IE, and skewing the div is a pain to maintain since it breaks different on different screen sizes. I have found that the safest way to do this (cross-browser) and keep the angle intact the best is to use these images.

Reply:
We will need to try to crash this then, because adding a second to the load time doesn't seem like a "safe" alternative. It could have negative effects on traffic.

Reply:
I think we can condense that size down in photoshop. we can compress it

Reply:
Let me see how much I can do

Reply:
At the very least, if we do have to use images can we potentially split it into multiple images.  JPG for the majority of the image, and PNG for just the part that needs the angle?

Reply:
^ Not sure how weird that will look